scripts/flash_device.py

```diff
@@ -0,0 +1,82 @@
+import subprocess32
+import time
+from argparse import ArgumentParser
+import os
+import get_latest_build
+
+
+def connect_device():
```

Comment:
We have this method in utilities. Why don't use it, instead. No cyclic dependencies afaik :D